internal/generated/snippets/secretmanager/apiv1/snippet_metadata.google.cloud.secretmanager.v1.json

```diff
@@ -0,0 +1,704 @@
+{
+  "clientLibrary": {
+    "name": "cloud.google.com/go/secretmanager/apiv1",
+    "version": "1.3.0",
+    "language": "GO",
+    "apis": [
+      {
+        "id": "google.cloud.secretmanager.v1",
+        "version": "v1"
+      }
+    ]
+  },
+  "snippets": [
+    {
+      "regionTag": "secretmanager_v1_generated_SecretManagerService_AccessSecretVersion_sync",
+      "title": "secretmanager AccessSecretVersion Sample",
+      "description": "AccessSecretVersion accesses a SecretVersion. This call returns the secret data.\n\nprojects/*/secrets/*/versions/latest is an alias to the most recently\ncreated SecretVersion.",
+      "file": "Client/AccessSecretVersion/main.go",
+      "language": "GO",
+      "clientMethod": {
+        "shortName": "AccessSecretVersion",
+        "fullName": "google.cloud.secretmanager.v1.Secretmanager.AccessSecretVersion",
```

Comment:
If the client is called `Client`, shouldn't this one here be `google.cloud.secretmanager.v1.Client.AccessSecretVersion`?